LanguageFeatures/Inline-classes/static_analysis_member_invocation_A02_t01.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Otherwise, a compile-time error occurs if V does not have a
+/// member named m.
```

Comment:
Just to make it explicit (because it could be easy to misunderstand this point from the text right here), `V` is an inline type and we're talking about members that `V` "has" (as opposed to "declares"). So when we get to `implements AnotherInlineClass, YetAnother` this includes members in the interface of `AnotherInlineClass` plus `YetAnother`.